source/MRMesh/MRPrism.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+#include "MRMeshFwd.h"
+
+namespace MR
+{
+    MRMESH_API Mesh makePrism( float baseLength, float leftAngle, float rightAngle, float height = 1.0f );
```

Comment:
please add comments about this function, prism orientation in space and triangle orientation, base position
also add [[nodiscard]] attrib to this function